en/docs/debug-the-code/debugging-sessions.md

```diff
@@ -1,7 +1,4 @@
-# Debugging overview
-The debugging experience of the Ballerina VS Code extension provides multiple options and features to debug Ballerina applications.
-
-## Debugging sessions
+# Debugging sessions
```

Comment:
Shouldn't this be `Debug sessions` @NipunaRanasinghe ?

Reply:
Done.